subnets/main.tf

```diff
@@ -14,3 +14,9 @@ resource "aws_subnet" "subnets" {
 
   tags = "${merge("${var.tags}",map("Name", "${var.project}-${var.visibility}-${var.role}-${element(data.aws_availability_zones.available.names, count.index)}", "Environment", "${var.environment}", "Project", "${var.project}", "Role", "${var.role}", "Visibility", "${var.visibility}"))}"
 }
+
+resource "aws_route_table_association" "subnet_association" {
+  count          = "${length(var.route_tables) >0 ? "${var.num_subnets}" : 0 }"
```

Comment:
What if I pass 1 route table and I have 3 subnets? Then count is 3 and on line 21 I'm exceeding the length of the `route_tables` list.